web/components/application/volunteer/pages/PageOne/PageOne.js

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+
+import { TextInput } from '../../../../input/text';
+import { HackerApplicationPageTemplate } from '../PageTemplate';
+import { Page } from '../../../Page';
+import { constraints } from './PageOneConstraints';
+import { SelectCity } from '../../../../input/select/SelectCity';
+import { Select } from '../../../../input/select';
+import { RadioButton, RadioGroup } from '../../../../input/buttons/RadioGroup';
+
+import { schools } from '../../../schools';
+import { genders } from '../../../genders';
+import { levelsOfEducation } from '../../../education';
+import { majors } from '../../../majors';
+import { ethnicities } from '../../../ethnicity';
+
+import books from '../../../../../assets/emoji/book.svg';
+import mountain from '../../../../../assets/emoji/mountain.svg';
+
+class PageOne extends Page {
+  constructor(props) {
+    super(props);
+
+    this.constraints = constraints;
+
+    const {
+      application: {
+        city,
+      },
+    } = this.props;
+
+    this.state.cityQuery = city;
+  }
+
+  isEducationChosen = education => education === levelsOfEducation[0]
+                                      || education === levelsOfEducation[1]
+                                      || education === levelsOfEducation[2]
+
+  isHighSchool = education => education === levelsOfEducation[0]
+
+  render() {
+    const {
+      application: {
+        firstName,
+        lastName,
+        email,
+        confirmEmail,
+        city,
+        school,
+        gender,
+        isOver19,
+        education,
+        gradYear,
+        major,
+        phoneNumber,
+        ethnicity,
+      },
+    } = this.props;
+
+    const {
+      cityQuery,
+    } = this.state;
+
+    return (
+      <HackerApplicationPageTemplate {...this.getPageTemplateProps()}>
+        <h1>About you <img className="emoji emoji-h1" alt="📚" src={books} /></h1>
+        <p>
+          {'nwHacks is Western Canada’s largest collegiate hackathon taking place on January 26th - 27th 2019 at the University of British Columbia. Join our team for 24 hours to get a snippet of what it’s like to organize a hackathon! Responsibilities include but not limited to registration, set-up/take-down, and more. Befriend some passionate hackers while distributing (and eating) midnight snacks. Volunteer hours are flexible and food & drinks will be provided.'}
+          <img className="emoji" alt="⛰️" src={mountain} />
+        </p>
+        <TextInput
+          label="What is your full legal name?"
+          placeholder="First Name"
+          name="first-name"
+          className="margin-ends-giga"
+          value={firstName}
+          onChange={(newFirstName) => {
+            this.updateApplication({
+              firstName: newFirstName,
+            });
+          }}
+          onBlur={() => this.setFieldAsBlurred('firstName')}
+          error={this.getErrorIfBlurred('firstName')}
+          />
+        <TextInput
```

Comment:
The `label` is missing for the last name input

Reply:
That's odd, I'm not sure how they would put in their full legal name if we only provide fields for first and last name. 🤔 I'll ping the logistics team about this

Reply:
I thought it was a bit strange too but ignored it due to the time crunch, let me know what they say!

Reply:
Anita says we can add the label back in
![image](https://user-images.githubusercontent.com/12876696/49187482-7efddb80-f31c-11e8-9656-23adfb378661.png)

Reply:
Fixing now !